Makefile

```diff
@@ -119,7 +119,7 @@ test:
 	@go test ${TESTARGS} `go list ./... | grep -v test/e2e`
 
 test-dependencies:
-	curl -L https://go.kubebuilder.io/dl/2.3.0/$(GOOS)/$(GOARCH) | tar -xz -C /tmp/
+	curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v$(KBVERSION)/kubebuilder_$(KBVERSION)_$(GOOS)_$(GOARCH).tar.gz | tar -xz -C /tmp/
```

Comment:
Shoot. This repos doesn't have the `KBVERSION` variable. You can either add it at the top or go back to hardcoding the version.